wolfcrypt/src/port/arm/armv8-sha256.c

```diff
@@ -1317,6 +1317,27 @@ int wc_Sha256Update(wc_Sha256* sha256, const byte* data, word32 len)
     return Sha256Update(sha256, data, len);
 }
 
+int wc_Sha256FinalRaw(wc_Sha256* sha256, byte* hash)
+{
+#ifdef LITTLE_ENDIAN_ORDER
+    word32 digest[WC_SHA256_DIGEST_SIZE / sizeof(word32)];
+#endif
+
+    if (sha256 == NULL || hash == NULL) {
+        return BAD_FUNC_ARG;
+    }
+
+#ifdef LITTLE_ENDIAN_ORDER
+    ByteReverseWords((word32*)digest, (word32*)sha256->digest,
```

Comment:
Do you need to double copy?

Reply:
Good suggestion. This is a copy/paste from https://github.com/wolfSSL/wolfssl/blob/master/wolfcrypt/src/sha256.c#L1194
I will update both places.

Reply:
That change could break some systems because casting the incoming byte* to word32* could result in a misalignment. Is it worth it to check the alignment of the pointer and avoid a double copy?

Reply:
Nope. Small amount of data copied, it want affect performance too much!